GETTING_STARTED.md

```diff
@@ -2,26 +2,27 @@
 
 1. Install [Node 16 LTS](https://nodejs.org/en/)
 2. Install and get familiar with [Git](https://docs.github.com/en/get-started/quickstart/hello-world)
```

Comment:
We should probably provide a link to the actual download site for Git.

Reply:
Followed up by #584.